### ISSUE TITLE
GUI 3 doesn't log same info to security log as GUI 1

### ISSUE THREAD

**Issue by jacodg** [ibissource/iaf#1336]
**Describe the bug**
On certain actions in the console (like changing log level) a message is written to the security log. When done in GUI 1 info is added about the remote host and remote user. GUI 3 doesn't add that information.

**Reporter**
Jaco de Groot

**Category** 
 - Frank Console

**To Reproduce**
1. Go to 'Environment Variables'
2. Change log level
3. View security log
4. See a line like:
2020-11-30 14:11:07,112 [WebContainer : 6] LogLevel changed from [DEBUG] to [ERROR]
5. Do the same in GUI 1 and see a line like:
2020-11-30 14:28:12,378 [WebControlShowEnvironmentVariables-post[WebContainer : 11]] LogLevel changed from [ERROR] to [DEBUG], logIntermediaryResults from [true] to [true] and logMaxMessageLength from [-1] to [-1] by remoteHost [localhost.localdomain] remoteAddress [127.0.0.1] remoteUser [******]

**Expected behavior**
GUI 3 should log the remote host/user information an all security events. Didn't investigate whether for some events GUI 3 does log this information. I think this needs to be investigated / might be clear when diving into the GUI 3 code. It would be nice to have an automated regression test to make sure the needed info is written to the log.

**Screenshots**
\-

**Environment:**
 - IAF 7.6-SNAPSHOT

**Additional context**
\-

**Comment by alisihab** [comment]
GUI 3 first logs the action( uri + method + user etc.) in the security log. Then the consequence of that action. For example, on my local env. I changed the log level from ERROR to DEBUG and IntermediaryResults from true to false.

> 2020-12-17 12:58:11,424 [http-nio-8088-exec-3] requestUri [/iaf/api/server/log] params [] method [PUT] remoteHost [0:0:0:0:0:0:0:1] remoteAddress [0:0:0:0:0:0:0:1] remoteUser [null]
2020-12-17 12:58:11,424 [http-nio-8088-exec-3] host[0:0:0:0:0:0:0:1] tried to access uri[/server/log] with origin[http://localhost:8088] but was blocked due to CORS restrictions
2020-12-17 12:58:11,432 [http-nio-8088-exec-3] LogLevel changed from [ERROR] to [DEBUG], logIntermediaryResults from [true] to [false]

There was an issue while logging some events and the initiator of those events. That was fixed with #1193
